README.md

```diff
@@ -196,6 +196,10 @@ spec:
   sampler:
     type: parentbased_traceidratio
     argument: "0.25"
+  python:
+    env:
+      - name: OTEL_EXPORTER_OTLP_ENDPOINT
```

Comment:
I think this deserves a comment (for us but as well for the users) to explain why this needs to be overridden.

There are as well other references in the code related to this issue: 

https://github.com/open-telemetry/opentelemetry-operator/blob/2ea988c6094e4d4a6d3c60571da2a516f895c9bb/autoinstrumentation/python/requirements.txt#L2

https://github.com/open-telemetry/opentelemetry-operator/blob/992b681dd1bb8445b2744fb4f602f0faa0128ac9/pkg/instrumentation/python.go#L60